src/meili/valhalla_run_map_match.cc

```diff
@@ -37,7 +37,7 @@ ReadMeasurements(istream_t& istream, float default_gps_accuracy, float default_s
 
 int main(int argc, char* argv[]) {
   if (argc < 2) {
-    std::cout << "usage: map_matching CONFIG" << std::endl;
+    std::cout << "usage: map_matching CONFIG" << '\n';
```

Comment:
there are a LOT of these. I'd rather not have these changes tbh. it's falling under "performance", but that's not really a concern big enough to have tidy fail IMO.

Reply:
I updated the code so `performance-avoid-endl` is not respected. quite a big change. changes with that option is still preserved in https://github.com/valhalla/valhalla/pull/5240/commits/632037310640fb11240b20a84f60f96f2bf0bf8e

Reply:
I don't see any changes in the `.clang-tidy` file. Can you double check that you've committed them?